### PR TITLE
ENT-4695 do not rotate keys on passive hub - 3.12.x

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom build HUB please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2878/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2878/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2878/

**Comment by Lex-2008** [comment]
@cf-bottom can you build it please?

**Comment by Lex-2008** [comment]
@vpodzime : please review, test, and merge after Jenkins done with the build

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4148/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4148/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4148/

**Comment by Lex-2008** [comment]
@vpodzime : probably we need a new build to test now that https://github.com/cfengine/masterfiles/pull/1588 is merged?

File: packaging/common/cfengine-hub/postinstall.sh [hunk 1]
```diff
@@ -891,7 +891,11 @@ else
     cf_console platform_service cfengine3 start
 fi
 
-if is_upgrade && wait_for_cf_postgres; then
+if cf-promises --show-classes | grep -q passive_ha_hub; then
```
Comment (vpodzime) [inline packaging/common/cfengine-hub/postinstall.sh]: This needs to use the absolute path.

**Comment by Lex-2008** [comment]
@cf-bottom can you build it, please?

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4187/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4187/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4187/

**Comment by vpodzime** [comment]
@Lex-2008 please take a look

**Comment by Lex-2008** [comment]
I'd approve this if only I wasn't the initial author of this PR,
so here's the _seal of approval_:
seal say yes

**Pull request merged**